### ISSUE TITLE
[L5R] Insight does not take into account Courtier/Etiquette 3 and 7 Mastery Abilities

### ISSUE THREAD

**Issue by kbernges** [Roll20/roll20-character-sheets#2227]
Each of the two skills provides the following bonuses to insight:
0, for 0<=rank<3
3, for 3<=rank<7
10, for rank>=7

I don't think this can be dealt with without conditionals, but it does have relevance if, for instance, a courtier character rather recently obtaining the next Insight Rank ending up in combat and needing to roll Initiative, which has Insight Rank (calculated from Insight) as part of the count of rolled dice.

Right now the only workaround is throwing ranks into a dummy skill equal to the amount of bonus to insight from the mastery abilities for these two skills.

**Comment by RhOmicron** [comment]
I'll see if I can add something for this. I guess I could just add either checkboxes or a options window which you choose the +3 or +10  insight bonus on them.

**Comment by RhOmicron** [comment]
Do the rules follow this? http://l5r-changing-history.obsidianportal.com/wikis/third-edition-revised-skill-list 

+2, +5, +7 etc?

**Comment by kbernges** [comment]
Older edition of the game. Current edition the masteries are 3, 5, and 7 for most skills that have them. I'll quickly type out the text from the book as an edit to this post.

**Comment by RhOmicron** [comment]
I'd like to fix this fully. I can easily add two boxes (which I have, that add +3 and +10. And are placed under insight). Would this be sufficent? Or do all skills still give + to insight?

**Comment by kbernges** [comment]
They're the only two that give Insight as Masteries. The Insight calculation is 1 Rank to 1 Insight, which is already accounted for on the sheet. This is one of those fun special exceptions that every system has.

**Comment by RhOmicron** [comment]
Right, I think I have everything fixed that you wanted then. I'll check it a bit more and make a pull request.

**Comment by RhOmicron** [comment]
#2237

**Comment by kbernges** [comment]
Your fix got broken by something in the last week. It was working well, but it among other things seem to have been broken by a pull request three days ago.

**Comment by RhOmicron** [comment]
Yes seem somebody that worked on something, just merged it right over the things I did.

**Comment by RhOmicron** [comment]
#2301
